modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -272,34 +271,123 @@ GEF<-function(setting,Forecast,Observed, H, extraArg, nitr=50000, nburnin=10000,
   #### from the interval matrix
   y.ind <- as.numeric(Y > interval[,1])
   y.censored <- as.numeric(ifelse(Y > interval[,1], Y, 0))
-
-  if(t == 1){ #TO DO need to make something that works to pick whether to compile or not
-    # Contants defined in the model
-    constants.tobit = list(N = ncol(X), YN = length(y.ind))
+  
+  #which type of observation do we have at this time point?
+  input.order <- lapply(input.vars, grep, x=names(obs.mean[[t]])) # not going to work if AbvGrnWood is given in two different ways like tree rings and refab
```

Comment:
If this won't work for conflicting proxies, do we want to go ahead and put in a better solution from the start?